.circleci/config.yml

```diff
@@ -16,6 +16,38 @@ defaults:
                   echo "${DOCKER_PASS}" | docker login -u="${DOCKER_USER}" --password-stdin
               fi
 jobs:
+  test:
+    docker:
+      - image: pypy:2.7
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+      # Future TODO: use a local dynamodb
```

Comment:
what do you mean a local one (isn't the docker localish?)

Reply:
Sort of. Ideally, I'd like to use https://hub.docker.com/r/circleci/dynamodb so we're not fetching an image, loading java, etc. I tried to get that working but kept hitting weird issues. Rather than rabbit hole on them, I just kept things as they were.